pkg/app/accounts/basic.go

```diff
@@ -35,6 +45,58 @@ func (m *HTTPBasicAccountManager) UserFromRequest(r *http.Request) (User, error)
 	return userFromRequest(r)
 }
 
+func (m *HTTPBasicAccountManager) UserLogin(w http.ResponseWriter, r *http.Request) error {
```

Comment:
What about separate the role between `GET` and `POST` request like below? I don't think there's a certain reason for handling `GET` and `POST` request in single function.

- `GET` : Visualize the input form and the output response from `POST` request
- `POST` : Try setting cookie with given input form from `GET` request, and return the result.

Reply:
Good suggestion! Done.